dockers/docker-orchagent/docker-init.j2

```diff
@@ -48,21 +48,23 @@ if [ "$SUBTYPE" == "DualToR" ]; then
     cp /usr/share/sonic/templates/tunnel_packet_handler.conf /etc/supervisor/conf.d/
 fi
 
+IS_SUPERVISOR=/etc/sonic/chassisdb.conf
 USE_PCI_ID_IN_CHASSIS_STATE_DB=/usr/share/sonic/platform/use_pci_id_chassis
 ASIC_ID="asic$NAMESPACE_ID"
-if [ -f "$USE_PCI_ID_IN_CHASSIS_STATE_DB" ]; then
-    while true; do
-        PCI_ID=$(sonic-db-cli -s CHASSIS_STATE_DB HGET "CHASSIS_ASIC_TABLE|$ASIC_ID" asic_pci_address)
-        if [ -z "$PCI_ID" ]; then
-            sleep 3
-        else
-            # Update asic_id in CONFIG_DB, which is used by orchagent and fed to syncd
-            if [[ $PCI_ID == ????:??:??.? ]]; then
-                sonic-db-cli CONFIG_DB HSET 'DEVICE_METADATA|localhost' 'asic_id' ${PCI_ID#*:}
-                break
+if [ -f "$IS_SUPERVISOR" ]; then
+    if [ -f "$USE_PCI_ID_IN_CHASSIS_STATE_DB" ]; then
+        while true; do
+            PCI_ID=$(sonic-db-cli -s CHASSIS_STATE_DB HGET "CHASSIS_ASIC_TABLE|$ASIC_ID" asic_pci_address)
```

Comment:
The original code apply to both IMM and supervisor.  But the new code is changed to only applicable to the supervisor card.  If it is for supervisor card, the key CHASSIS_ASIC_TABLE should be CHASSIS_FABRIC_ASIC_TABLE.  But this PR requires the sonic-platform-daemon #331.